src/staticarrays_support.jl

```diff
@@ -0,0 +1,10 @@
+import StaticArrays: StaticArray, tuple_prod
+
+@generated function StructArrays.staticschema(::Type{s}) where {s <: StaticArray{S,T,N}} where {S,T,N}
+    return quote
+        Base.@_inline_meta
+        return NTuple{$(tuple_prod(S)),T}
+    end
+end
+StructArrays.createinstance(::Type{T}, args...) where {T<:StaticArray} = T(args...)
```

Comment:
```suggestion
StructArrays.createinstance(::Type{T}, args...) where {T<:StaticArray} = T(args)
```

From what I understand of the [interface](https://github.com/JuliaArrays/StaticArrays.jl/blob/master/src/StaticArrays.jl#L55), all implementations are required to take a flat tuple, rather than many arguments.